src/08_example/04_multithreading.md

```diff
@@ -0,0 +1,26 @@
+# Adding Parallelism
+Our example so far has largely presented concurrency (using async code)
+as an alternative to parallelism (using threads).
+However, async code and threads are not mutually exclusive.
+Async executors can be either single-threaded or multithreaded.
+For example, the [`async-executor` crate](https://docs.rs/async-executor) used by `async-std`
+has both a single-threaded `LocalExecutor` and a multi-threaded `Executor`.
+
+Tasks do not always need to be run on the thread that created them,
```

Comment:
nit: Neutral value judgment here, something like "tasks can run either on the same thread that created it, or a different thread."